lib/kubernetes-deploy/runner.rb

```diff
@@ -282,10 +283,12 @@ def validate_namespace
     def run_kubectl(*args, namespaced: true, with_context: true)
       args = args.unshift("kubectl")
       if namespaced
+        raise FatalDeploymentError, "Namespace missing for namespaced command" if @namespace.blank?
         args.push("--namespace=#{@namespace}")
       end
 
       if with_context
+        raise FatalDeploymentError, "Explicit context is required to run this command" if @context.blank?
```

Comment:
These are errors that people will see in deploy logs right? I suspect "namespace" and especially "context" might be too ambiguous as a term for most. "kubectl config context" maybe? too long?